corehq/apps/users/views/mobile/users.py

```diff
@@ -669,6 +675,223 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class MobileWorkerListView(JSONResponseMixin, BaseUserSettingsView):
+    template_name = 'users/mobile_workers.html'
+    urlname = 'mobile_workers'
+    page_title = ugettext_noop("Mobile Workers")
+
+    @method_decorator(use_bootstrap3())
+    @method_decorator(use_select2())
+    @method_decorator(require_can_edit_commcare_users)
+    def dispatch(self, *args, **kwargs):
+        return super(MobileWorkerListView, self).dispatch(*args, **kwargs)
+
+    @property
+    def can_bulk_edit_users(self):
+        if not user_can_edit_any_location(self.request.couch_user, self.request.project):
+            return False
```

Comment:
These first two lines don't need to be here - I think we copy/pasted that from the locations version of this